Hardware/LPC/NCT677X.cs

```diff
@@ -414,7 +415,7 @@ public NCT677X(Chip chip, byte revision, ushort port, LPCPort lpcPort)
     public float?[] Controls { get { return controls; } }
 
     private void DisableIOSpaceLock() {
-      if (chip != Chip.NCT6791D)
+      if (chip != Chip.NCT6791D || chip != Chip.NCT6793D)

```

Comment:
100%, as it stands the condition will always be true.  `chip` would have to be both `NCT6791D` and `NCT6793D` at the same time in order to fail the condition!